File: bddtests/src/main/kotlin/mocking/vision/models/appointments/BookedAppointmentsResponse.kt
package mocking.vision.models.appointments

import mocking.vision.models.appointments.BookedAppointmentsResponse.Companion.name
import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(namespace = "urn:vision", name = name)
@XmlAccessorType(XmlAccessType.FIELD)
data class BookedAppointmentsResponse(
        @XmlElement(namespace = "urn:vision", name = "slot")
        var slots: Slots? = null,
        @XmlElement(namespace = "urn:vision")
        var references: References? = null,
        @XmlElement(namespace = "urn:vision", name = "settings")
        var settings: AppointmentSettings? = null
) {

    companion object {
        const val name = "bookedAppointmentsResponse"
    }
}

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Vision/Models/BookedAppointmentsResponse.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Vision.Models
{
    [XmlType(TypeName = "bookedAppointmentsResponse", Namespace = "urn:vision")]
    public class BookedAppointmentsResponse
    {
        [XmlElement(ElementName = "bookedAppointmentsResponse", Namespace = "urn:vision")]
        public BookedAppointments Appointments { get; set; }
    }
}
